velocity_planner/velocity_planner/triangle.py

```diff
@@ -1,7 +1,7 @@
+# What is this? Why does this exist? Where is this used
```

Comment:
It's the same as trapezoidal planner but it doesn't gradually ramp down, it just instantly stops. I'll delete it.